UtilityBeltTests/UtilityBeltDataTests/Extensions/XCTestCase+MeasureAndCatch.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+extension XCTestCase {
```

Comment:
This might not be necessary with the new error mechanics of Xcode 11.4's testing functionality, but it matches the implementation for `UtilityBeltKeychain` so I'm going to leave it for now.